### DISCUSSION TITLE
Geo data for GB Testcase

### DISCUSSION THREAD

**Discussion by FrancisMujjuni** [e2nIEE/pandapower#1143] [Q&A]
I am using Pandapower and especially the data about GBReducednetwork. The geo coordinates (x and y) in the data frame bus_geodata are not in any known coordinate reference system that would place the components in the UK. How would you advise me to convert them, say, to epsg:4326.

**Comment by rbolgaryn** [comment]
can you provide the code to load the network you are using?
